DECISION-RECORD.md

```diff
@@ -7,4 +7,97 @@ Wel loggen we de error.
 
 **Soms weten we wel al in welk zaaksysteem een bepaald gegeven te vinden is. Bij de communicatie tussen front-end en backend volgen we zoveelmogelijk de standaard api's. Hoe communiceren we extra informatie over het bronsysteem zonder van de api's af te wijken?**
 We kiezen ervoor een header mee te sturen.
-Het alternatief zou zijn, om deze informatie helemaal te negeren en altijd elk gegeven in alle beschikbare zaaksystemen op te zoeken. De belangrijkste overweging om dit niet te doen is, naast het grote aantal redundante calls, de complexiteit die ontstaat in het correct afhandelen van fouten en niet gevonden gegevens.
\ No newline at end of file
+Het alternatief zou zijn, om deze informatie helemaal te negeren en altijd elk gegeven in alle beschikbare zaaksystemen op te zoeken. De belangrijkste overweging om dit niet te doen is, naast het grote aantal redundante calls, de complexiteit die ontstaat in het correct afhandelen van fouten en niet gevonden gegevens.
+
+## Zaken bij bedrijven nieuwe stijl
```

Comment:
ik mis hier wat context. wat is de nieuwe stijl, welk probleem levert dat op, wat zijn de opties?